pkg/registry/core/shoot/strategy.go

```diff
@@ -209,13 +210,13 @@ func (shootStrategy) AllowUnconditionalUpdate() bool {
 }
 
 // WarningsOnCreate returns warnings to the client performing a create.
-func (shootStrategy) WarningsOnCreate(ctx context.Context, obj runtime.Object) []string {
-	return nil
+func (s shootStrategy) WarningsOnCreate(ctx context.Context, obj runtime.Object) []string {
+	return shoot.GetWarnings(ctx, obj.(*core.Shoot), nil, s.credentialsRotationInterval)
 }
 
 // WarningsOnUpdate returns warnings to the client performing the update.
-func (shootStrategy) WarningsOnUpdate(ctx context.Context, obj, old runtime.Object) []string {
-	return nil
+func (s shootStrategy) WarningsOnUpdate(ctx context.Context, obj, old runtime.Object) []string {
```

Comment:
Can you overwrite this for `shootStatusStrategy`? 
Otherwise, we will flood gardenlet logs with warnings from status updates probably?